lib/dispatcher.js

```diff
@@ -4,7 +4,12 @@ Router.prototype.dispatcher = function () {
   var trie = this.trie
 
   return function* trieRouter (next) {
-    var match = trie.match(this.request.path)
+    try {
+      var match = trie.match(this.request.path)
+    } catch (err) {
+      err.code = 1101;
```

Comment:
we should do something more human readable. ex. https://github.com/stream-utils/raw-body/blob/66f380f89b1975b3e6c670faa8ebdb67919652ee/index.js#L270